lib/lbt/bundle/BundleDefinition.js

```diff
@@ -27,5 +27,13 @@ export const SectionType = {
 	 * Usually used as the last section in a merged module to enforce loading and
 	 * execution of some specific module or modules.
 	 */
-	Require: "require"
+	Require: "require",
+
+	/**
+	 * Dependency cache information that lists modules and their dependencies
+	 * of all types: JS, declarative views/fragments.
+	 * Only the dependencies of the modules are stored as 'depCache' configuration.
+	 * Requires UI5 Evolution runtime.
```

Comment:
Can be removed. It requires ui5loader (since UI5 1.54), which also applies to all other APIs currently used. Legacy API support has been removed with v3.0 (see https://github.com/SAP/ui5-builder/pull/739).
```suggestion
```